aikau/src/main/resources/alfresco/preview/PdfJs/DocumentView.js

```diff
@@ -309,10 +310,10 @@ define(["dojo/_base/declare",
          {
             var currentPage = this.pages[0],
                 container = currentPage.container,
-                hmargin = parseInt(domStyle.get(container, "margin-left")) + parseInt(domStyle.get(container, "margin-right")),
-                vmargin = parseInt(domStyle.get(container, "margin-top")),
-                contentWidth = parseInt(currentPage.content.pageInfo.view[2]),
-                contentHeight = parseInt(currentPage.content.pageInfo.view[3]),
+                hmargin = parseInt(domStyle.get(container, "margin-left"), 10) + parseInt(domStyle.get(container, "margin-right"), 10),
+                vmargin = parseInt(domStyle.get(container, "margin-top"), 10),
+                contentWidth = parseInt(currentPage.content.pageInfo.view[2], 10),
+                contentHeight = parseInt(currentPage.content.pageInfo.view[3], 10),
```

Comment:
The decision when I introduced JSHint was that we didn't want to enforce JSHint yet, only have it in an advisory role. It might be worth reconsidering, but there's still the problem that there are over 1,000 JSHint warnings that we would need to cleanup first. Once we were in that position though, we could certainly put JSHint into Bamboo and run that on commit.